package.json

```diff
@@ -15,7 +15,9 @@
         "test:watch": "TZ=UTC vitest",
         "tsc": "tsc --noEmit --skipLibCheck",
         "analyze": "cross-env ANALYZE=true pnpm prod",
-        "i18n:sync": "sync-i18n --files 'lang/*.json' --primary en_US --languages en_GB es_ES pl_PL pt_BR --space 4"
+        "i18n:rename-to-dashes": "renamer --find \"_\" --replace \"-\" \"lang/*.json\"",
+        "i18n:rename-to-underscores": "renamer --find \"-\" --replace \"_\" \"lang/*.json\"",
```

Comment:
The sync script uses filenames to auto-recognize the source/target locales. This presents a challenge: in the Laravel ecosystem, locales are conventionally written as "en_US". But in the Node.js ecosystem, they are conventionally written as "en-US".

Before the sync command runs, quickly rename all the files to the Node.js standard, and then after the sync command is done, name them back.

The files should remain using the Laravel convention, as it is plausible there will be strings returned from the back-end that need to be localized.